pkgs/development/interpreters/bqn/cbqn/default.nix

```
@@ -103,7 +104,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/dzaima/CBQN/";
     description = "BQN implementation in C";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ AndersonTorres sternenseemann synthetica shnarazk ];
+    maintainers = with maintainers; [ AndersonTorres sternenseemann synthetica shnarazk detegr ];
```

```suggestion
    maintainers = with maintainers; [ AndersonTorres sternenseemann synthetica shnarazk ];
```

Makes no sense to maintain only the interpreter but not the bytecode.

---

Added myself as maintainer for bytecode package as well.